prow/tide/tide.go

```diff
@@ -345,7 +345,8 @@ func (c *Controller) pickBatch(sp subpool) ([]PullRequest, error) {
 func (c *Controller) mergePRs(sp subpool, prs []PullRequest) error {
 	for _, pr := range prs {
 		if err := c.ghc.Merge(sp.org, sp.repo, int(pr.Number), github.MergeDetails{
-			SHA: string(pr.HeadRef.Target.OID),
+			SHA:         string(pr.HeadRef.Target.OID),
+			MergeMethod: c.ca.Config().Tide.MergeMethod(sp.org + "/" + sp.repo),
```

Comment:
If the consumer has `org` and `repo`, expose that as the API to for `MergeMethod()` and do the transform inside of the call

Reply:
I was trying to keep the separation of concerns here. If the `MergeMethod()` excepts an org and a repo shouldn't the data be stored that way in the yaml? But, the yaml doesn't hold to that pattern today.

Reply:
This isn't a huge divergence from the config representation and the API doesn't need to correspond to the representation in the config exactly. It more important to have an easy to use API.